packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_title_card.dart

```diff
@@ -37,7 +41,7 @@ class KnowledgePanelTitleCard extends StatelessWidget {
               iconUrl: knowledgePanelTitleElement.iconUrl,
               width: 36,
               height: 36,
-              color: colorFromEvaluation,
+              color: iconColorFromEvaluation,
```

Comment:
`color: colorFromEvaluation,`

Reply:
Changes done, will be updated in upcoming commit.